docusaurus/docs/using-https-in-development.md

```diff
@@ -31,3 +31,9 @@ HTTPS=true npm start
 ```
 
 Note that the server will use a self-signed certificate, so your web browser will almost definitely display a warning upon accessing the page.
+
+You can also specify a custom certificate (available in `react-scripts@2.2.0` and higher).
```

Comment:
Need to bump the version - does `3.1.0` work?
```suggestion
You can also specify a custom certificate (available in `react-scripts@3.1.0` and higher).
```

Reply:
Yes, @iansu how would you want to handle this? I think 3.1.0 sounds good to me.

Reply:
@mrmckeb @iansu Should we go with `3.1.0` and get this merged, or is there other stuff we need to wait for?